homeassistant/components/androidtv/media_player.py

```diff
@@ -490,15 +495,7 @@ async def async_get_media_image(self):
     @adb_decorator()
     def get_raw_media_data(self):
         """Raw base64 image data."""
-        try:
-            response = self.aftv.adb_shell("screencap -p | base64")
-        except UnicodeDecodeError:
-            return None
-
-        if isinstance(response, str) and response.strip():
-            return binascii.a2b_base64(response.strip().replace("\n", ""))
-
-        return None
+        return self.aftv.adb_screencap()
```

Comment:
This description is no longer accurate, it's raw PNG data.

Reply:
Raw PNG data is what the function returned before ... the PNG > raw was done in the function :P ... just changed the comment 👍